### ISSUE TITLE
Tab code-completion does not work in terminal input against screen readers

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#113482]
CC @isidorn

## System Info

Version: 1.53.0-insider (system setup)
Commit: 4a875e23d20b64504a818834f3fa4c40adb8d480
Date: 2020-12-21T12:33:38.481Z
Electron: 11.1.0
Chrome: 87.0.4280.88
Node.js: 12.18.3
V8: 8.7.220.29-electron.0
OS: Windows_NT x64 10.0.19042

NVDA does not read code suggestion or tab completion in terminal input area (Ctrl+`) for bash shell, etc.

For example, please kindly try out with [radian console](https://github.com/randy3k/radian).

I think this is due in part to the fact that the terminal input has inheritted "document" role from its parent container: The "document" role should be applied to the terminal output region only.

**Comment by meganrogge** [comment]
Duplicate of #98918
